component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpSource.java

```diff
@@ -214,7 +214,8 @@ public void disconnect() {
 
     @Override
     public void destroy() {
-        this.httpConnectorRegistry.stopHttpServerConnectorController();
+        // TODO: 7/26/17 Until fix for multiple worker and boss thread loop group
+        //this.httpConnectorRegistry.stopHttpServerConnectorController();
```

Comment:
@Yasara123 When can we expect this fix? If it's not going to happen any time soon, better to have a GIT issue created for this, so we will not miss to fix this.